### ISSUE TITLE
Launchpad: Gate the new Launchpad task list to Automatticians for testing purposes

### ISSUE THREAD

**Issue by valterlorran** [Automattic/wp-calypso#77273]
Similarly to https://github.com/Automattic/wp-calypso/issues/77271 we should gate the new Launchpad task list to Automatticians for testing purposes.

This implementation relies on having the `is_enabled` callback for the task list that https://github.com/Automattic/wp-calypso/issues/77270 should introduce.

More context about this discussion here: p1684324340423669/1684262724.806499-slack-C0Q664T29

**Comment by valterlorran** [comment]
We probably want to include this check in the `wpcom_is_launchpad_keep_building_enabled` hook linked below:

fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Szh%2Qcyhtvaf%2Sjcpbz%2Qynhapucnq.cuc%3Se%3Qn046764r%2312-og

```php
function wpcom_is_launchpad_keep_building_enabled( $is_enabled ) {
	return is_serenity_team_member() && is_automattic();
}
add_filter( 'is_launchpad_keep_building_enabled', 'wpcom_is_launchpad_keep_building_enabled' );
```

**Comment by sixhours** [comment]
Started working on this in D113526-code

**Comment by sixhours** [comment]
Closed by D113526-code
